TypeCobol.Test/Parser/Programs/Cobol85/ExecSql/ExecSqlWithExecuteImmediateStatement.rdzSQL.txt

```diff
@@ -0,0 +1,18 @@
+﻿--- Diagnostics ---
+Line 7[14,20] <27, Error, Syntax> - Syntax error : The host variable mustn't have any indicator variable RuleStack=codeElement>executeImmediateStatement,  OffendingSymbol=[14,20:EXECUTE]<SQL_EXECUTE>
```

Comment:
Could you change the message to match the DB2 spec closer: `An indicator variable must not be specified with a host variable in EXECUTE IMMEDIATE statement`